src/app/models/category.go

```diff
@@ -2,17 +2,16 @@ package models
 
 import (
 	"time"
-
-	_ "github.com/jinzhu/gorm/dialects/mysql"
-	validator "gopkg.in/go-playground/validator.v9"
 )
 
 type (
 	Category struct {
-		Id    int    `json:"id"`
-		Name  string `json:"name"`
-		Slug  string `json:"slug"`
-		Posts []Post `json:"posts" gorm:"many2many:post_category;"`
+		Id      int       `json:"id"`
```

Comment:
__[golint]__
* [golint] struct field Id should be ID

<!-- Comment by [SideCI](https://sideci.com) -->